azure-pipelines.yml

```diff
@@ -531,6 +531,8 @@ jobs:
   - bash: |
       set -e
       . util/build_consts.sh
+      module load "xilinx/vivado/$(VIVADO_VERSION)"
+      updatemem --help # TODO: remove before merging
```

Comment:
TODO: remove before merging

Reply:
I saw the expected output in CI logs. I'll probably update and merge this PR to unblock splicing in CI. Thanks @milesdai